website/redirects.js

```diff
@@ -201,8 +201,8 @@ const Redirects=[
   },
   {
     "from": [
-      "/docs/latest/ingestion/flatten-json.html",
-      "/docs/latest/ingestion/transform-spec.html"
+      "/docs/latest/ingestion/transform-spec.html",
+      "/docs/latest/ingestion/flatten-json.html"
     ],
     "to": "/docs/latest/ingestion/ingestion-spec"
```

Comment:
Should we split this into two entries? Since the Markdown file redirected to `/docs/latest/ingestion/data-formats`, we might want to keep that. But keep redirecting `transform-spec` to `ingestion-spec`.

Also, let's double-check the flatten-json redirection works as expected (maybe something is off, which would be why the markdown was added a couple of weeks ago?).